<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detect</class>
 <widget class="QDialog" name="Detect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Person:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>500</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recognize">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>500</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Recognize</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>6</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_cam">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Cam is off! Please turn it on!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="startCam">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>StartCam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="takePic">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TakePIC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changePerson">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>500</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ChangePerson</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>face1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>face4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>face2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>face3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>face5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>180</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>face6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>face7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>face8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label9">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>face9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePic">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SavePIC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trainData">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>460</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TrainData</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Detect</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
